packages/ssi-sdk-core/src/utils/image.ts

```diff
@@ -1,25 +1,67 @@
 import fetch from 'cross-fetch'
-import sizeOf from 'image-size'
-import { ISizeCalculationResult } from 'image-size/dist/types/interface'
+import { imageSize } from 'image-size'
 import { IImageDimensions, IImageResource } from '../types'
 
+type SizeCalculationResult = {
+  width?: number
+  height?: number
+  orientation?: number
+  type?: string
+}
+
+const uint8ArrayToString = (uint8Array: Uint8Array): string => {
```

Comment:
Please use the uint8arrays package for that as we are using that in everywhere. It has a toString method with encoding support

Reply:
changed to use that library